src/ContextMenu.tsx

```diff
@@ -115,22 +114,35 @@ const menuStyles = (theme: ITheme['contextMenu'], hasMenuItems: boolean) => {
  * MENUITEM
  */
 
-export interface IContextMenuItem extends Omit<IBox, 'onClick'> {
-  title?: string;
-  data?: Object;
-  divider?: boolean;
-  disabled?: boolean;
-  preventClose?: boolean;
-  onClick?: (
-    event: React.TouchEvent<HTMLDivElement> | React.MouseEvent<HTMLDivElement>,
-    data: Object,
-    target: HTMLElement
-  ) => void | Function;
-  menuItems?: IContextMenuItem[];
-}
+type ContextMenuTitle = { title?: string; shortcut?: never } | { title: string; shortcut?: string };
```

Comment:
In case you're wondering what this stuff is about — is to make the type checker aware that if I supply a `shortcut`, then the `title` is mandatory, otherwise it's not. @P0lip 

We should do this more often.

Reply:
Yeah, I like `never` same as `unknown`. Used  `unknown` on a number of occasions lately.